ImplementationGuide/markdown/UebergreifendeFestlegungen/UebergreifendeFestlegungen_Must-Support-Flags.md

```diff
@@ -1,13 +1,34 @@
 ## Must-Support-Flags
 
-Die Verwendung des [Must-Support-Flags](https://hl7.org/fhir/R4/profiling.html#mustsupport) an Profil-Elementen hat im Kontext dieses Leitfadens folgende Bedeutung:
+Das [Must-Support-Flags](https://hl7.org/fhir/R4/profiling.html#mustsupport) (MS-Flag) kennzeichnet Elemente, die auf bestimmte Weise unterstützt werden müssen. Sowohl für die Erstellung - d.h. das Exponieren der Profile -, als auch für die Verarbeitung - d.h. den Umgang beim Ausgang von extern - greifen die MS-Flag Festlegungen. 
+Die Verwendung des MS-Flag an Profil-Elementen hat im Kontext dieses Leitfadens folgende Bedeutung:
 
 1. Im Kontext der Erstellung von FHIR-Ressourcen:
 
-    Wenn die entsprechende Information in einem der Testszenarien bekannt ist, dann MUSS das Feld in der erstellten Ressource gefüllt sein. Ist die Information nicht bekannt, KANN es leer bleiben.
+    Elemente, die mit MS gekennzeichnet sind, MÜSSEN vom erstellenden System implementiert werden. Dies bedeutet:
+    - Systeme MÜSSEN die entsprechenden Elemente in der eigenen Persistenz-Ebene vorhalten können. 
+    - Systeme MÜSSEN die entsprechenden Elemente in den entsprechenden Profil-Instanzen bei Abfrage über eine FHIR-Schnittstelle befüllen und ausgeben können.
+    - Systeme MÜSSEN die entsprechenden Elemente zur Anzeige bringen können (ausgenommen davon sind Systeme mit reiner Middleware-Funktion, die über keine grafische Nutzeroberfläche verfügen).
+    -  MS-Elemente, die als optional gekennzeichnet sind (0..-Kardinalität) KÖNNEN bei der Erstellung einer FHIR-Instanz fehlen, wenn die entsprechende Information im Einzelfall unbekannt, nicht zutreffend ist oder (noch) nicht erhoben wurde.
+    - Wenn MS-Elemente fehlen, SOLLEN Systeme über den Grund des Fehlens Informationen bereitstellen.
```

Comment:
Das ist unklar. Klingt, als müssten die das für jede Instanz machen, in der ein optionales MS-Element fehlt! Obeohl einen Satz drüber steht, dass das kein Problem ist. Du meinst, dass sie Informationen bereitstellen sollen, warum sie ein MS nicht implementieren können/wollen, oder?

Reply:
Offengesagt war das ein SHortcut, um diese Vorgabe zu verallgemeinern : "When information on a particular data element is missing or suppressed, refer to the guidance for [Missing Data](https://build.fhir.org/ig/HL7/US-Core/general-requirements.html#missing-data) and [Suppressed Data](https://build.fhir.org/ig/HL7/US-Core/general-guidance.html#suppressed-data). In cases where information on a specific data element is missing, and the US Core Responder knows the precise reason for the absence of data (other than suppressed data), US Core Responders SHOULD send the reason for the missing information. This is done by following the same methodology outlined in the [Missing Data](https://build.fhir.org/ig/HL7/US-Core/general-requirements.html#missing-data) section but using the appropriate reason code instead of unknown." https://build.fhir.org/ig/HL7/US-Core/must-support.html

Reply:
@simoneOnFhir hast du einen besseren Vorschlag? Bist du dafür zu streichen?

Reply:
Würde ich komplett weglassen. Wenn das MS Element optional ist, dann sollen sie's einfach weglassen und gut. Ich finde das unnötig kompliziert und halte es für unrealistisch, dass das so implementierbar wäre...